### ISSUE TITLE
Minimized libraries?

### ISSUE THREAD

**Issue by DJXAKEP** [ziahamza/webui-aria2#165]
Why you don't use minimized versions of js/css libraries (jquery, angular, bootstrap, etc)?

**Comment by ziahamza** [comment]
Yeah, it was mostly a convinience while developing the product. A build
script would be really helpful to minimize the entire project into one
html, css and JS file.
